bin/make_cfgs.py

```diff
@@ -34,6 +34,11 @@
     INSERT INTO nodes (name, machine_type, mac_address, is_vm, locked, arch, description)
         VALUES  ( 'ses-client-3.a.b.de' , 'client' ,
             'aa:bb:cc:dd:67:16' , 'f' , 'f' , 'x86-64' , '' ) ;
+
+also generated udev rules file for predefined network names setup
+    #ses-client-1
+    SUBSYSTEM=="net", ACTION=="add", DRIVERS=="?*", ATTR{address}=="aa:bb:cc:dd:67.16", ATTR{dev_id}=="0x0", ATTR{type}=="1", KERNEL=="eth*", NAME="eth0"
```

Comment:
Is the hardcoded mac there by intention?

Reply:
it is just an example, no more, mac is not real